### PR TITLE
User Story 3: "Reset History"

### PR DESCRIPTION
**Pull request by mndys**
## Value Proposition (User Story)
As a **user**,
I need to **reset my history**
because I want to **start over**
## Description (Text, Scribble, Design)
User Story 3
## Acceptance Criteria (Edge Cases)
- [x] history is cleared
- [x] history state hook is an empty array
## Tasks
- [x] Create instance of Button as reset button
- [x] Add logic to update history state hook
- [x] Add localStorage for history
- [x] Write test for History component
- [x] Write test for localStorage
## Story Points (Complexity)
M

### CODE REVIEW

File: src/components/App.spec.js [hunk 1]
```diff
@@ -10,6 +11,24 @@ it('renders a Button with textContent "Spin!"', () => {
   render(<App />)
   expect(screen.getByRole('button', { name: 'Spin!' })).toBeVisible()
 })
-it.todo('History component is invisible as long as history state hook is empty')
-it.todo('history array holds up to 10 prompts before it demands to be emptied')
-it.todo('the same prompt is not given twice')
+
+it('writes to localStorage once on spin button click', () => {
```
Comment (eileenfuerstenau) [inline src/components/App.spec.js:15]: Very cool testing! I learned something here

File: src/components/App.jsx [hunk 2]
```diff
@@ -5,52 +5,76 @@ import Header from './Header'
...
+    setHistory([])
+    saveToLocal('promptHistory', [])
+    setCurrentPrompt(INITIALPROMPT)
+    saveToLocal('currentPrompt', INITIALPROMPT)
+  }
+
+  function onSpin() {
+    let randomPrompt = prompts[getRandomNumber()]
+
     function getRandomNumber() {
```
Comment (eileenfuerstenau) [inline src/components/App.jsx:56]: You might want to source out this relatively long function to a separate file, to keep your App.js more clear?

**Review by eileenfuerstenau — COMMENTED** [review]
Great job! You thought of some very good solutions & manage to keep things understandable

File: src/components/App.jsx (hunk 2 above)
Comment (mndys) [reply src/components/App.jsx:56]: good idea!! Thank you very much! 😊

**Pull request merged**